distribution/sdk/ddf-sso-query-widget/src/main/java/org/codice/ddf/ui/Query.java

```diff
@@ -254,41 +252,24 @@ private String getProxyTicket(HttpServletRequest request) {
     }
 
     /**
-     * 
-     * @param unformattedXml
-     *            Unformatted xml.
+     * @param unformattedXml Unformatted xml.
      * @return Returns formatted xml.
      */
     private String format(String unformattedXml) {
         Source xmlInput = new StreamSource(new StringReader(unformattedXml));
-        StringWriter stringWriter = new StringWriter();
-        StreamResult xmlOutput = new StreamResult(stringWriter);
-        TransformerFactory transformerFactory = TransformerFactory.newInstance();
+        String formattedXml;
 
-        Transformer transformer = null;
-        String formattedXml = null;
-
-        try {
-            transformer = transformerFactory.newTransformer();
-            LOGGER.debug("transformer class: {}", transformer.getClass());
-            transformer.setOutputProperty(OutputKeys.INDENT, "yes");
-            transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
-            transformer.transform(xmlInput, xmlOutput);
-            formattedXml = xmlOutput.getWriter().toString();
-        } catch (TransformerConfigurationException e) {
-            String message = "Unable to transform xml:\n" + unformattedXml
-                    + "\nUsing unformatted xml.";
-            LOGGER.error(message, e);
-            formattedXml = unformattedXml;
-        } catch (TransformerException e) {
-            String message = "Unable to transform xml:\n" + unformattedXml
-                    + "\nUsing unformatted xml.";
-            LOGGER.error(message, e);
-            formattedXml = unformattedXml;
-        }
+        TransformerProperties transformProperties = new TransformerProperties();
+        transformProperties.addOutputProperty(OutputKeys.INDENT, "yes");
```

Comment:
This is the most popular set of properties so far.  Should we create a `prettyPrint` or `prettyFormat` method that uses these settings by default?